odoo/addons/base/wizard/base_export_language.py

```diff
@@ -35,18 +35,34 @@ def _get_languages(self):
     state = fields.Selection([('choose', 'choose'), ('get', 'get')], # choose language or get the file
                              default='choose')
 
+    def _reopen(self):
+        return {
+            'name': self.env.ref('base.action_wizard_lang_export').name,
+            'type': 'ir.actions.act_window',
+            'res_model': 'base.language.export',
+            'view_mode': 'form',
+            'res_id': self.id,
+            'views': [(False, 'form')],
+            'target': 'new',
+        }
+
+    def act_retry(self):
+        this = self[0]
```

Comment:
I don't like `this` in python. You can use `self.ensure_one()` or just ignore the batch things. Because I don't thing a wizard model will be called in batch